contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/getters.js

```diff
@@ -5,19 +5,47 @@ import uniqBy from 'lodash/uniqBy';
 
 import { validateNodeDetails, validateNodeFiles } from 'shared/utils/validation';
 import { ContentKindsNames } from 'shared/leUtils/ContentKinds';
+import { RolesNames } from 'shared/leUtils/Roles';
+import { NEW_OBJECT } from 'shared/constants';
 
 function sorted(nodes) {
   return sortBy(nodes, ['lft']);
 }
 
-export function getContentNode(state) {
+export function getContentNode(state, getters) {
   return function(contentNodeId) {
     const node = state.contentNodesMap[contentNodeId];
     if (node) {
       const thumbnail_encoding = JSON.parse(node.thumbnail_encoding || '{}');
       const tags = Object.keys(node.tags || {});
+      const aggregateValues = {};
+      if (node.kind === ContentKindsNames.TOPIC) {
+        const children = getters.getContentNodeChildren(contentNodeId);
+        for (let child of children) {
+          aggregateValues['error_count'] =
+            (aggregateValues['error_count'] || 0) +
+            (child['error_count'] || Number(child['complete']));
```

Comment:
I'm not sure I understand the `Number(child['complete'])`? Should it be `Number(!child['complete'])`?

Reply:
i confirm this, without it the warning still apear

Reply:
Yep, missing the bang!